documentation/meta/documentation/adding_docs.md

```diff
@@ -0,0 +1,34 @@
+# Adding a new documentation page
+
+This is a quick guide for adding a new documentation page and running the
+documentation locally.
+
+## Prerequisites
+
+Refer to the [general setup guide](/general/general_setup.md) for setting up the
+prerequisites. Refer to the 'Docs' column in the
+[requirement matrix](/general/general_setup.md#requirement-matrix) to know what
+you need to run this.
+
+## Starting up
+
+The following demonstrates the process of adding a new documentation page.
+
+1. Create a markdown file at a place within the `documentation/` hierarchy that
+   makes the most sense for your case, this would usually already contain a
+   `index.md` file, make sure that your new file is located next to it.
+
+   For instance, adding a new documentation file `dag_testing.md` for describing
+   how to test/run the catalog DAGs would likely have you adding the file in
+   `documentation/catalog/guides/`.
+
+2. Make an entry with the name of the new document(excluding the `.md` file
```

Comment:
Nit:
```suggestion
2. Make an entry with the name of the new document (excluding the `.md` file
```

Reply:
Awesome suggestions @AetherUnbound. Would make the changes and revert.

Reply:
PR has been updated with suggested changes.